fonduer/parser/parser.py

```diff
@@ -269,13 +283,117 @@ def _parse_figure_node(self, node, state):
 
         return state
 
-    def _parse_sentence(self, node, state):
+    def _parse_sentence(self, paragraph, node, text, field, state):
```

Comment:
I'm not a big fan of how many arguments this function has. Any suggestions on simplifying this would be welcome. I don't want to have to alter the `state`.

Reply:
Can we put text and field into state? Then when we done with _parse_sentence we can remove those two.

Reply:
Yes. Good idea.